elasticsearch/templates/participants_structured.json

```diff
@@ -287,6 +287,46 @@
             }
           }
         },
+        "files": {
```

Comment:
@pegahtah @SimoneMaiwald : File answers will have the "file upload guid" as the `answer` value. The guid corresponds to a "file" object in this list. This object should have all you need.

* `bucket` and `blobName` is what you need to access the file from storage.
* There's metadata like `fileName`, `fileSize`, `mimeType` that you can display to study staff.
* `scanResult` can be `null`, `CLEAN`, `INFECTED`.
* `uploadedAt`, `scannedAt`, and `scanResult` will be `null` if file hasn't been scanned for malware yet. In this case, we should avoid allowing study staff to access file since it's not safe.
* Likewise, if `scanResult` is `INFECTED` I urge us NOT to allow study staff to access file in any way.

Reply:
Here's an example:
```
"files": [
  {
    "guid":"flEdlDLBN54tsfqsQxnm",
    "bucket":"ddp-dev-file-scanned",
    "blobName":"testboston/F2X6R0E76Q56A65N3B3X/BASELINE_COVID/flEdlDLBN54tsfqsQxnm",
    "fileName":"yufeng.pdf",
    "fileSize":50580,
    "mimeType":"application/pdf",
    "authorizedAt":"2021-02-09T17:34:14.942414Z",
    "uploadedAt":"2021-02-09T17:34:30.546Z",
    "scannedAt":"2021-02-09T17:35:57.273Z",
    "scanResult":"CLEAN"
  }
],
```

Reply:
files is part of a pt in the structured_participant index, correct?

Reply:
Yup! It's a top-level property in the `participants_structured` index.